.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "extensions/src/ACRE2Profiling/tracy"]
```

Comment:
instead of submodules maybe just use cmake fetch

```
include(FetchContent)
FetchContent_Declare(
    tracy	
    GIT_REPOSITORY https://github.com/wolfpld/tracy
    GIT_TAG v0.9.1
)
FetchContent_MakeAvailable(tracy)
```